perf/benchmark/run_benchmark_job.sh

```diff
@@ -111,6 +111,46 @@ function setup_fortio_and_prometheus() {
     export FORTIO_SERVER_POD
 }
 
+function port_forward_envoy() {
+  kubectl -n "${NAMESPACE}" port-forward "${1}" 15000 &
+}
+
+function kill_envoy_port() {
+    # shellcheck disable=SC2155
+    kill $(lsof -t -i:15000)
+}
+
+function collect_config_dump() {
+  kill_envoy_port
+  port_forward_envoy "${FORTIO_CLIENT_POD}"
+  FORTIOCLIENT_CONFIG_DUMP="${FORTIO_CLIENT_POD}"_config_dump.yaml
```

Comment:
This is a lot of repeated code in 2 fns, refactor?

```bash
function collect (url, dump_dir, file_name_suffix) {
}
```
and then
``` bash
collect "/config_dump" "config_dumps" _config_dump.yaml 
```

Reply:
yeah, sure!